projects/ad4630_fmc/zed/README.md

```diff
@@ -37,81 +37,96 @@ The overwritable parameters from the environment are:
 - DDR_EN: in echo and master clock mode, the SDI lines can have Single or Double data rates
   - 0 - MISO runs on SDR (default)
   - 1 - MISO runs on DDR
+- NO_REORDER: removes the spi_axis_reorder from system for CAPTURE_ZONE = 1 and
+  NUM_OF_SDI = 1 (ad4030) or NUM_OF_SDI = 2 (ad4630) and directly connects the SPI
+  Engine to DMA
+  - 0 - spi_axis_reorder present (default)
+  - 1 - spi_axis_reorder removed
 
 ### Example configurations
 
-#### Clock mode SPI, MOSI lines 4, Capture zone 2, SDR (default)
+#### Clock mode SPI, MISO lines 4, Capture zone 2, SDR (default)
 
 This specific command is equivalent to running `make` only:
 
 ```
-make CLK_MODE=0 NUM_OF_SDI=4 CAPTURE_ZONE=2 DDR_EN=0
+make CLK_MODE=0 NUM_OF_SDI=4
```

Comment:
Why?

Reply:
CAPTURE_ZONE=2 DDR_EN=0 are default parameters for build, has no effect if you add them explicitly at make

Reply:
Same comment here as well as below